kontrakter/politi/innsettelsesordre/1.0/eksempelfiler/innsettelsesordre-paatale.json

```diff
@@ -480,56 +482,34 @@
     ]
   },
   "ordreEllerKjennelseVaretekt": {
```

Comment:
Sendes alle lovbud i siktelsen som en liste?
Hvordan ivaretar vi koblinga mellom de ulike lovbud og tilhørende krimtype og liste av statistikkgruppe?
Så det var noe ferdig struktur i "straffeloven.json"... 

Eksempel på use case internt i KO: 
Gitt lovbud + Krimtype + statistikkgruppe kan gi konsekvens at man har 3 betjenter tilstede. 
Kombinert med annen lovbud + krimtype + statistikkgruppe som gir annen ekstra konsekvens. Osv.

Reply:
@JosteinKlakegg, det Martin gjør er å liste opp alle straffesakene i politiets sakskompleks. 

Hovedstraffesaksnummeret peker på straffesaken med hovedlovbudet.  (Dvs. den "viktigste" saken i komplekset, hvis saksnummer vi bruker som identifikasjon når vi snakker om saken/sakskomplekset på tvers av etatene).

Vi i KO vil i første omgang fokusere på krimtype m.m. fra hovedstraffesaken. Men det at vi nå får detaljene for *alle* straffesakene åpner for at vi kan bygge mer avansert logikk og statistikk i framtiden.


Hver straffesak har en krimtype+statistikkgruppe. Den er koblet til et lovbrudd, en liste med medsiktede, fornærmede, etc. Den inkluderer politiets *hjemmelsrekke* og strafferammen på denne. (Merk at *hjemmelsrekka* er mer enn en referanse til kun ett område i en av dagens lover på Lovdata.. En hjemmelsrekka som inkluderer §79 (mafiaparagrafen) kan f.eks. forhøye strafferammen til det dobbelte av hva strafferammen normalt er for brudd på en gitt paragraf.) Spør meg hvis vi skal ta mer detaljer muntlig :)